### PR TITLE
dev-to-alpha

### PR DESCRIPTION
**Pull request by szuecs**
* **Adding manifests for kubernetes-lifecycle-metrics**
   <sup>Merge pull request #1543 from zalando-incubator/kube-lifecycle-metrics</sup>
* **enable clusterRatelimits**
   <sup>Merge pull request #1567 from zalando-incubator/enable/clusterRatelimit</sup>

### CODE REVIEW

File: cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      application: kubernetes-lifecycle-metrics
+  template:
+    metadata:
+      labels:
+        application: kubernetes-lifecycle-metrics
+        version: master-1
+      annotations:
+        kubernetes-log-watcher/scalyr-parser: '[{"container": "kubernetes-lifecycle-metrics", "parser": "json"}]'
+    spec:
+      priorityClassName: system-cluster-critical
```
Comment (szuecs) [inline cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml:22]: Do we really need this for all components that only provide metrics?
IMHO that should not be done, but more for the important things like metric server that cause an outage of HPA or skipper that cause an outage of http routing.

Comment (aermakov-zalando) [reply cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml:22]: Why? I don't see why the users' pods should be able to preempt infrastructure stuff.

Comment (szuecs) [reply cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml:22]: If we have only limited CPU/memory and we have to select the important and the unimportant workloads, then I would say skipper-ingress or Prometheus is more important than this. I think we should use different priorities for different workloads, not treat all the same, does it makes sense?

Comment (aermakov-zalando) [reply cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml:22]: It does, but I don't think sure it's *that* important. We always have the users' pods to sacrifice, and it won't kick out anything important, so I see no harm in leaving it as is.

**Comment by szuecs** [comment]
:+1:

File: cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml (hunk 1 above)
Comment (linki) [reply cluster/manifests/kubernetes-lifecycle-metrics/deployment.yaml:22]: @szuecs @aermakov-zalando good to go?

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**